h2o-algos/src/main/java/hex/schemas/TreeV3.java

```diff
@@ -0,0 +1,30 @@
+package hex.schemas;
+
+import water.Iced;
+import water.api.API;
+import water.api.schemas3.KeyV3;
+import water.api.schemas3.SchemaV3;
+
+public class TreeV3 extends SchemaV3<Iced, TreeV3> {
+    @API(required = true, direction = API.Direction.INPUT, help = "Key of the model the desired tree belongs to",level = API.Level.critical)
+    public KeyV3.ModelKeyV3 model;
+
+    @API(required = true, direction = API.Direction.INPUT, help = "Index of the tree in the model.", level = API.Level.critical)
+    public int tree_number;
+
+    @API(direction = API.Direction.INPUT, help = "Tree class index. Equals to 1 for regression/binomial. For multinomial, ranges from 0 to response column cardinality - 1.", level = API.Level.critical)
+    public int tree_class;
```

Comment:
Maybe we should use domain value instead of the index.

Reply:
We are doing it now. Tested.